lib/cog/repository/bundles.ex

```diff
@@ -39,6 +39,8 @@ defmodule Cog.Repository.Bundles do
   @install_types [:normal, :force]
   @default_install_type :normal
 
+  @minimum_bundle_config_version Spanner.Config.minimum_config_version
```

Comment:
I added some comments @kevsmith. Were those not clear enough? Or is Github just not showing the changes?